app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, old_name, directory = command.split()
```

Comment:
You are using `_` right below this line. So better give it a name. Cause we usually name variables as `_` when we are sure that we never ever gonna use it